src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -246,29 +278,63 @@ Item {
                     onCountChanged: { scrollToEnd(); }
                     delegate: Rectangle {
                         id: nameDelegate
```

Comment:
This delegate becomes overwhelming, 400++ lines already :/ could/should we move it to a different file?